docs/releases/v0.9.0.md

```diff
@@ -1,18 +1,24 @@
 # Changelog
 
-## [v0.9.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.9.0) (2021-04-27)
+## [v0.9.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.9.0) (2021-04-28)
```

Comment:
Can you rebase the code? This PR now includes Min's change. It's hard to review.

Reply:
yeah i submit a new pr and merged the correct version from mainline, problem resolved